tox.ini

```diff
@@ -31,10 +31,14 @@ setenv =
     devdeps,build_docs,online: HOME = {envtmpdir}
     HIDE_PARFIVE_PROGESS = True
     NO_VERIFY_HELIO_SSL = 1
+    devdeps: PIP_EXTRA_INDEX_URL = https://pkgs.dev.azure.com/astropy-project/astropy/_packaging/nightly/pypi/simple/ https://pypi.anaconda.org/scipy-wheels-nightly/simple
+pip_pre =
+    devdeps: true
+    !devdeps: false
```

Comment:
The only issue I can imagine is that it might pull in more devdeps than we expect from pypi?

Reply:
This tox option just adds `--pre` to the pip command. I'd need to check if that applies to the dependencies of explicitly requested packages also. It only has an effect when installing the tox reps and definitely has no effect when installing sunpy.

Reply:
> The only issue I can imagine is that it might pull in more devdeps than we expect from pypi?

This might be a good thing? Perhaps we can see how it goes for a bit, and we get more errors than expected have a re-think?

Reply:
It is installing pre-releases of astropy and matplotlib dependents: `greenlet`, `numpy`, `pandas` and `scipy`. And the pre-release `numpy` seems to be causing the test fails.

Reply:
If its a pre-release numpy causing the issue, I feel like we should inform someone upstream?